src/streaming/models/VideoModel.js

```diff
@@ -293,6 +293,14 @@ function VideoModel() {
         return element ? element.videoHeight : NaN;
     }
 
+    function getVideoRelativeOffsetTop() {
+        return element && element.parentNode ? element.offsetTop - element.parentNode.offsetTop : NaN;
+    }
+
+    function getVideoRelativeOffsetLeft() {
+        return element && element.parentNode ? element.getBoundingClientRect().x - element.parentNode.getBoundingClientRect().x : NaN;
```

Comment:
@jeffcunat,Looking at MDN (https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) seems object returned by getBoundClientRect() doesn't include x/y properties in Edge/IE/Safari.